scripts/omGeoJSON.py

```diff
@@ -42,48 +57,65 @@ def processGeoJSON():
     # Load domain
     print("Loading output file")
     ds = nc.Dataset(domainOutputPath)
-    landmask = ds["LANDMASK"][:]
-    lats = ds["LATD"][:][0]
-    longs = ds["LOND"][:][0]
-    ch4 = ds["OCH4_TOTAL"][:][0][0]
-    maxEmission = np.amax(ch4)
+
+    # There is a better way to do this but this will work for now
+    # Using xarray wasn't straightforward because the layers don't use
+    # the same attribute names, ie mixing x/y and lat/long.
+    ds_slice = {
+        "LANDMASK": ds["LANDMASK"][:][0],
+        "LATD": ds["LATD"][:][0][0],
+        "LOND": ds["LOND"][:][0][0],
+    }
+
+    max_values = {}
+    for layer_name in prior_layers:
+        # extract the meaningful dimensions from the NetCDF variables
+        ds_slice[layer_name] = ds[layer_name][:][0][0]
+        # find the max emission value in a single cell for each layer
+        max_values[layer_name] = np.amax(ds_slice[layer_name])
```

Comment:
```suggestion
        max_values[layer_name] = np.max(ds_slice[layer_name])
```
Max and amax are the same. I had to look up what amax was haha

Reply:
Haha all good, it was like that when I found it!

Updated in 2c7edba.